dmcontent/utils.py

```diff
@@ -98,3 +99,14 @@ def get_option_value(option):
     :return: string value to be persisted
     """
     return option.get('value') or option['label']
+
+
+def try_load_manifest(content_loader, application, data, question_set, manifest):
+    try:
+        content_loader.load_manifest(data['slug'], question_set, manifest)
+
+    except ContentNotFoundError:
+        # Not all frameworks have this, so no need to panic (e.g. G-Cloud 4, G-Cloud 5, G-Cloud-6)
```

Comment:
The comment doesn't make sense in this context.

Reply:
Stupid comment...